app/src/main/java/ar/com/wolox/android/cookbook/scanqr/scanmenu/ScanMenuFragment.kt

```diff
@@ -0,0 +1,32 @@
+package ar.com.wolox.android.cookbook.scanqr.scanmenu
+
+import ar.com.wolox.android.cookbook.R
+import ar.com.wolox.android.cookbook.scanqr.ScanQrActivity
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+import kotlinx.android.synthetic.main.fragment_scan_menu.*
+import javax.inject.Inject
+
+class ScanMenuFragment @Inject constructor() : WolmoFragment<ScanMenuPresenter>(), ScanMenuView {
+
+    override fun layout(): Int = R.layout.fragment_scan_menu
+
+    override fun init() {
+        presenter.attachView(this)
+        setListeners()
+    }
+
+    override fun setListeners() {
+        super.setListeners()
+        vScanQrButton.setOnClickListener {
+            presenter.onScanClicked()
+        }
+
+        vGenerateQrButton.setOnClickListener {
+            // TODO GENERATE QR
```

Comment:
You will add this on another PR?

Reply:
This was a task that Body was going to do but then he left 😓

Reply:
:sad-deleted-person:

Reply:
:rip-kb: